xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/screensaver.h

```diff
@@ -8,6 +8,9 @@
 
 #pragma once
```

Comment:
Why not drop `#pragma once` now that we have include guards?

Reply:
Have removed the `#pragma once` as them really no more needed, thanks for the note about.